scripts/release-on-comment.py

```diff
@@ -168,6 +171,22 @@ def trigger_release(payload_path: Path, token: str) -> None:
         )
         print_and_exit(f"{Fore.RED}{e}")
 
+    if error_contents:
+        link = f"https://github.com/{SLUG}/actions/runs/{os.environ['GITHUB_RUN_ID']}"
+        issue.create_comment(
+            f"""
```

Comment:
This is missing a `dedent()` call (and has a dangling `)` below for that reason).

Reply:
fixed.